stock-tracker/src/features/header/component/marketStatus.tsx

```diff
@@ -28,7 +29,7 @@ export const MarketStatus = () => {
   return (
     <div className="market_status_display">
       <p className="market_status_display__real_time">{realTimeDisplay}</p>
-      <img className="market_status__icon" src={statusIcon} />
+      <img className="market_status__icon" src={statusIcon} alt="" />
```

Comment:
If you can, don't bypass this attribute - <img> elements want an "alt" attribute in case something's wrong (download failed, corrupt image, or even for a blind person).

Some sensible values for this would be "open" or "close" based on response.isUSMarketOpen